### PR TITLE
[feat]: Add to wishlist component done

### PR DESCRIPTION
**Pull request by raghav2110**
Hi , I have added wishList component in my ecom. Please review it.

### CODE REVIEW

File: src/backend/db/products.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ export const products = [
     avalQty: 10,
     brand: "Nike",
     categoryName: "men",
+    inCart: false,
+    inWishlist: false,
```
Comment (abhij1607) [inline src/backend/db/products.js:23]: I implemented it without adding this specification in backend products, not sure which approach is better

File: src/context/auth-context.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { itemReducer } from "../utilities/items/itemReducer";
+
+const itemSettings = {wishList:[]};
+
+const AuthContext = createContext(itemSettings);
+
+const useAuth = () => useContext(AuthContext);
+
+const AuthProvider = ({ children }) =>{
+    const [wishList, setWishList] = useState([]);
```
Comment (abhij1607) [inline src/context/auth-context.jsx:11]: is this really required?

**Review by abhij1607 — COMMENTED** [review]
Great work done || Overall clean code

Left a comment for the suggestion, the rest looks good and can be merged

File: src/backend/db/products.js (hunk 1 above)
Comment (shivampandey0) [reply src/backend/db/products.js:23]: Same confusion as @abhij1607.

**Review by shivampandey0 — COMMENTED** [review]
Overall good code, the product is added from the product listing page and being removed from the wishlist page. All is done in context as of now and can be later extended to use API calls for this.
Good to merge

File: src/backend/db/products.js (hunk 1 above)
Comment (raghav2110) [reply src/backend/db/products.js:23]: Will look at another solution as well.

**Pull request merged**